media/js/mozorg/about-transparency.js

```diff
@@ -0,0 +1,31 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+(function($, Mozilla) {
+    'use strict';
+
+    var reportSections = $('#report-sections');
+
+    if (reportSections) {
+        if (typeof matchMedia !== 'undefined') {
+
+            var queryWide = matchMedia('(min-width: 760px)');
+
+            if (queryWide.matches) {
+                Mozilla.Accordion.destroyAccordionById('report-sections');
+            } else {
+                reportSections = new Mozilla.Accordion(reportSections);
```

Comment:
This line is throwing errors and causing odd accordion behavior when squishing/expanding the browser, but it's only because you're assigning the accordion to the same variable you're using to create the accordion. `Mozilla.Accordion` expects an element, which it's getting the first time, but if you expand the contract again, `reportSections` has been converted to an instance of `Mozilla.Accordion`, which causes problems.

Easy fix though - just call `new Mozilla.Accordion(reportSections)` (here and below). No need to assign the result to a variable.